src/store/modules/milk/reducer.ts

```diff
@@ -1,4 +1,4 @@
-import { createReducer } from '../../../utils/CoreModule';
+import { createReducer, CoreModule } from '../../../utils/CoreModule';
 import { MilkEntry } from './types';
 
-export default createReducer<MilkEntry>('milk');
+export default createReducer<MilkEntry>(CoreModule.MILK);
```

Comment:
why did you choose to use milk rather than the Dairy reducer we already have?

Reply:
I had milk and Bea made a good point on my other PR that cheese is also dairy, and our naming conventions should be specific to what we are doing. Plus I think it's called milk in the database

Reply:
I just did it. the only problem is all the selectors depended on a list of milk entries to make their calculations but there isn't a list of entries that I can find in milk

Reply:
What do you mean? it should be `state.milk.rows` I think?